src/libstore/binary-cache-store.cc

```diff
@@ -183,6 +183,9 @@ ref<const ValidPathInfo> BinaryCacheStore::addToStoreCommon(
            compression == "lzip" ? ".lzip" :
            compression == "lz4" ? ".lz4" :
            compression == "br" ? ".br" :
+           compression == "gzip" ? ".gz" :
+           compression == "lzma" ? ".lzma" :
+           compression == "compress" ? ".compress" :
```

Comment:
What format is "compress"?

Reply:
I don't have a need for it, but did found that it works. Seems to produce archives that can be manipulated with: https://linux.die.net/man/1/uncompress

I came across the possibility of that option here: https://github.com/NixOS/nix/blob/53857552278173641c0489a3196c51aab769bac8/src/libutil/compression.cc#L265

Reply:
Maybe that one should be “.Z” to be standard.